src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -79,6 +81,9 @@ public abstract class PeerConnector : IPeerConnector
 
         /// <summary>The network the node is running on.</summary>
         private Network network;
+        
+        /// <summary>Peers connected to this peer.</summary>
+        protected IPEndPoint[] connectedEndpoints;
```

Comment:
Is this still needed? We have a property/field called `ConnectorPeers`?

Reply:
You are correct I can replace this with ConnectorPeers.  Have removed it.